README.md

```diff
@@ -8,9 +8,16 @@ Blobfuse2 is stable, and is ***supported by Microsoft*** provided that it is use
 [This](https://github.com/Azure/azure-storage-fuse/tree/main?tab=readme-ov-file#config-guide) section will help you choose the correct config for Blobfuse2.
 
 ##  NOTICE
-- If you are using versions 2.2.0, 2.2.1 and 2.3.0, refrain from using Block-cache mode and switch to `file-cache` mode till [known issues](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2-Known-issues) are fixed.
+- If you are using versions 2.2.0, 2.2.1 and 2.3.0, refrain from using Block-cache mode and switch to `file-cache` mode. [Known issues](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2-Known-issues) in these versions are fixed in version **`2.3.2`**.

```

Comment:
We should ask customers to upgrade as the primary action. Reworded statement below

To avoid encountering [known issues](https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2-Known-issues) when using the 'Block-cache' mode, users of Blobfuse versions 2.2.0, 2.2.1, and 2.3.0 are advised to promptly upgrade to version 2.3.2.